### DISCUSSION TITLE
Run daemon after install

### DISCUSSION THREAD

**Discussion by fedex03** [transmission/transmission#6253] [Q&A] [answered]
Hi there,

installed transmission following these [instructions ](https://github.com/transmission/transmission#building-a-transmission-release-from-the-command-line).

Everything seems ok but I don't know where to find the config file or start the daemon.

I didn't find anything on the guide.

Thanks.

**Comment by tearfur** [comment, marked as answer]
The install location is defined by [CMAKE_INSTALL_PREFIX](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html), which is `/usr/local` by default.

The location of the config files are documented at [here](https://github.com/transmission/transmission/blob/main/docs/Configuration-Files.md)
